recipes/ngseqbasic/meta.yaml

```diff
@@ -0,0 +1,38 @@
+
+package:
+  name: ngseqbasic
+  version: 2.0.1
+
+source:
+  fn: NGseqBasic.tar.gz
+  url: https://github.com/telenius/NGseqBasic/raw/master/NGseqBasic.tar.gz
+  md5: 3992a5c4ed260e548db9b47cad3009be
+
+build:
+  skip: True # [osx]
+  number: 0
+
+requirements:
+  run:
+    - samtools ==0.1.19
+    - bowtie ==1.0.0
+    - bowtie2 ==2.3.0
+    - bedtools ==2.17.0
+    - ucsc-bedgraphtobigwig ==332
+    - ucsc-bedclip ==332
+    - ucsc-bedgraphpack ==332
+    - ucsc-bedtobigbed ==332
+    - flash ==1.2.11
+    - trim-galore ==0.4.1
+    - perl ==5.22
```

Comment:
@rvalieris it is failing when I use perl instead of perl-threaded:
https://travis-ci.org/CGATOxford/bioconda-recipes/jobs/202614745#L547

could you please tell why?